src/NUnitFramework/framework/Internal/Commands/OneTimeSetUpCommand.cs

```diff
@@ -65,15 +65,8 @@ public override TestResult Execute(TestExecutionContext context)
             if (_typeInfo != null)
             {
                 // Use pre-constructed fixture if available, otherwise construct it
-                if (!_typeInfo.IsStaticClass)
-                {
-                    context.TestObject = _suite.Fixture ?? _typeInfo.Construct(_arguments);
-                    if (_suite.Fixture == null)
-                    {
-                        _suite.Fixture = context.TestObject;
-                    }
-                    Test.Fixture = _suite.Fixture;
-                }
+                if (!_typeInfo.IsStaticClass && context.TestObject == null)
+                    context.TestObject = _typeInfo.Construct(_arguments);
```

Comment:
Just started investigating this. If I remove `&& context.TestObject == null` the currently failing tests all pass. Unfortunately, over 50 tests that are passing start to fail. I'll keep delving.